client/src/components/controls/TimelineWrapper.vue

```diff
@@ -19,6 +19,9 @@ export default defineComponent({
       :maxFrame="mediaController.maxFrame.value"
       :frame="mediaController.frame.value"
       :seek="mediaController.seek"
+      :time="mediaController.time.value"
+      :volume="mediaController.volume.value"
+      :setVolume="mediaController.setVolume"
```

Comment:
TimelineWrapper should probably be  deleted.  It's here as a bridge between `injectMediaController` which you need composition api for, and the controls, which we hadn't re-written in composition now.

You could move the injection into `ControlsContainer.vue` and get rid of this altogether.